tests/base/src/test/java/org/keycloak/tests/admin/authentication/ProvidersTest.java

```diff
@@ -85,12 +86,12 @@ public void testClientAuthenticatorProviders() {
                 "Validates client based on signed JWT issued by client and signed with the Client private key", false);
         addClientAuthenticatorProviderInfo(expected, "client-secret", "Client Id and Secret", "Validates client based on 'client_id' and " +
                 "'client_secret' sent either in request parameters or in 'Authorization: Basic' header", true);
-        addClientAuthenticatorProviderInfo(expected, "testsuite-client-id-required", "Signed Jwt", "Validates client based on signed JWT issued by client " +
-                "and signed with the Client private key", false);
-        addClientAuthenticatorProviderInfo(expected, "testsuite-client-passthrough", "Testsuite Dummy Client Validation", "Testsuite dummy authenticator, " +
-                "which automatically authenticates hardcoded client (like 'test-app' )", false);
-        addClientAuthenticatorProviderInfo(expected, "testsuite-client-dummy", "Testsuite ClientId Dummy",
-                "Dummy client authenticator, which authenticates the client with clientId only", false);
+//        addClientAuthenticatorProviderInfo(expected, "testsuite-client-id-required", "Signed Jwt", "Validates client based on signed JWT issued by client " +
+//                "and signed with the Client private key", false);
+//        addClientAuthenticatorProviderInfo(expected, "testsuite-client-passthrough", "Testsuite Dummy Client Validation", "Testsuite dummy authenticator, " +
+//                "which automatically authenticates hardcoded client (like 'test-app' )", false);
+//        addClientAuthenticatorProviderInfo(expected, "testsuite-client-dummy", "Testsuite ClientId Dummy",
+//                "Dummy client authenticator, which authenticates the client with clientId only", false);
```

Comment:
Is this stuff something we don't have in the framework yet and will be added in the future?

Reply:
Going to remove the commented out lines. Was supposed to do that once I got the test passing, and verified the providers where custom, but forgot. Will remove the comments now.